src/server/common/model/state/state-manager.js

```diff
@@ -24,21 +24,27 @@ export class StateManager {
 
   /** @returns {RawApplicationState} */
   toState() {
-    return Object.fromEntries(
-      sectionKeys
-        .map((key) => [key, this._request.yar.get(key)])
-        .filter((entry) => entry.at(1) !== null)
+    return (
+      Object.fromEntries(
+        sectionKeys
+          .map((key) => [key, this._request.yar.get('application')?.[key]])
+          .filter((entry) => entry.at(1) !== null)
+      ) || {}
```

Comment:
I think we can just return `this._request.yar.get('application') ?? {}`?

Reply:
and remove `sectionKeys` as a result.

Reply:
done